fastai/tabular/data.py

```diff
@@ -12,22 +12,42 @@ class TabularDataLoaders(DataLoaders):
     "Basic wrapper around several `DataLoader`s with factory methods for tabular data"
     @classmethod
     @delegates(Tabular.dataloaders, but=["dl_type", "dl_kwargs"])
-    def from_df(cls, df, path='.', procs=None, cat_names=None, cont_names=None, y_names=None, y_block=None,
-                valid_idx=None, **kwargs):
-        "Create from `df` in `path` using `procs`"
+    def from_df(cls,
+        df:pd.DataFrame,
+        path:(str,Path)='.', # Location of `df`, defaults to current working directory
+        procs:list=None, # List of `TabularProc`s
+        cat_names:list=None, # Column names pertaining to categorical variables
+        cont_names:list=None, # Column names pertaining to continuous variables
+        y_names:list=None, # Names of the dependent variables
+        y_block:TransformBlock=None, # `TransformBlock` to use for the target(s)
+        valid_idx:list=None, # List of indices to use for the validation set, defaults to a random split
+        **kwargs
+    ):
+        "Create `TabularDataLoaders` from `df` in `path` using `procs`"
         if cat_names is None: cat_names = []
         if cont_names is None: cont_names = list(set(df)-set(L(cat_names))-set(L(y_names)))
         splits = RandomSplitter()(df) if valid_idx is None else IndexSplitter(valid_idx)(df)
         to = TabularPandas(df, procs, cat_names, cont_names, y_names, splits=splits, y_block=y_block)
         return to.dataloaders(path=path, **kwargs)
 
     @classmethod
-    def from_csv(cls, csv, skipinitialspace=True, **kwargs):
-        "Create from `csv` file in `path` using `procs`"
+    def from_csv(cls,
+        csv:(str,Path,io.BufferedReader), # A csv from which features/targets are inferred
```

Comment:
This isn't accurate. 

```suggestion
        csv:(str,Path,io.BufferedReader), # A csv of training data
```

Reply:
Done ✨